src/components/story-elements/brightcove.js

```diff
@@ -68,12 +90,27 @@ const BrightcoveElement = (props) => {
       getPoster();
     }
     return (
-      <div className="brightcove-wrapper">
+      <div className="brightcove-wrapper" ref={videoRef}>
         {!showVideo && (
           <>
-            <button className="brightcove-playBtn" onClick={() => loadLibrary()} aria-label="Play Video" />
+            <button
+              className="brightcove-playBtn"
+              onClick={() => {
+                loadLibrary();
+                setThumbnailClicked(true);
+              }}
+              aria-label="Play Video"
+            />
             {posterImage ? (
-              <img className="brightcove-poster" onClick={() => loadLibrary()} src={posterImage} alt="video" />
+              <img
+                className="brightcove-poster"
+                onClick={() => {
+                  loadLibrary();
+                  setThumbnailClicked(true);
+                }}
+                src={posterImage}
+                alt="video"
+              />
```

Comment:
the onclick can be added to the container directly

Reply:
removed onclick event repeatition